yt_astro_analysis/radmc3d_export/RadMC3DInterface.py

```diff
@@ -110,13 +110,19 @@ class RadMC3DWriter:
     included in the radiative transfer calculation by radmc3d:
 
     >>> import yt
+    >>> import unyt as un
     >>> from yt.extensions.astro_analysis.radmc3d_export.api import RadMC3DWriter
 
     >>> x_co = 1.0e-4
-    >>> mu_h = yt.Quantity(2.34e-24, 'g')
+    >>> mu_h = un.unyt_quantity(2.34e-24, 'g')
     >>> def _NumberDensityCO(field, data):
     ...     return (x_co/mu_h)*data["Density"]
```

Comment:
Maybe this one, too?
```suggestion
    ...     return (x_co/mu_h)*data["gas", "density"]
```